.travis.yml

```diff
@@ -3,12 +3,7 @@ jdk:
   - openjdk7
 os:
   - linux
-cache:
-  directories:
-    - ~/.m2/repository
 install:
-  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true --quiet --batch-mode --show-version --activate-profiles travis
-before_script:
-  - echo "MAVEN_OPTS=${TEST_MEM_SETTINGS}" > ~/.mavenrc
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -P travis
 script:
-  - mvn cobertura:cobertura coveralls:report --batch-mode --activate-profiles travis -DrepoToken=${COVERALLS_REPO_TOKEN} -Dtest.arguments="${TEST_MEM_SETTINGS}"
+  - mvn clean test cobertura:cobertura coveralls:report -B -P travis -DrepoToken=${COVERALLS_REPO_TOKEN}
```

Comment:
Same here. The test.arguments can go though, I assume that was copied over from CT but we shouldn't have memory issues here.